src/util/fees.cpp

```diff
@@ -28,8 +30,8 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-const std::map<std::string, FeeEstimateMode>& FeeModeMap() {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
+const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap() {
+    static const std::vector<std::pair<std::string, FeeEstimateMode>> fee_modes = {
```

Comment:
could squash this commit (eda34393d51c59ace68e08df2d6e00870fe5d797)?

Reply:
Squashed.